music-app/src/components/LikedSongs.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Song from './Song.js';
+import { getlikedSongs } from '../actions';
+
+const token = localStorage.getItem('token');
+
+class LikedSongs extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      songs: [],
+    };
+  }
+
+  componentDidMount() {
+    this.props.getlikedSongs(token);
+  }
+
+  render() {
+    this.props.songs && console.table(this.props.songs.songs[1]);
+    return (
+      <div>
+        <h1>Liked Songs Dashboard</h1>
+        {this.props.songs.songs.map(song => (
+          <Song song={song} />
+        ))}
+        {/* // {this.props.songs.songs[1] && this.props.songs.songs[1].track.name} */}
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  songs: state.likedSongsReducer,
```

Comment:
finish mapping state to props by referencing which part of the likedSongsReducer state you're connecting to, 
change line 35 to : 
```
songs: state.likedSongsReducer.songs
```
Going back to my comments on the above lines we won't have to have such a long props call and it should be just:
```
this.props.songs
```